doc/syntax.md

```diff
@@ -2,42 +2,323 @@
 
 NESTML has a concise syntax that avoids clutter in the form of
 semicolons, curly braces or tags as known from other programming or
-description languages. Instead it concentrates on the domain concepts
+markup languages. Instead it concentrates on the domain concepts
```

Comment:
We could also point this out in the first sentence: "NESTML is a domain specific language which has a concise..."